@here/harp-mapview/lib/poi/PoiRenderer.ts

```diff
@@ -459,15 +458,21 @@ export class PoiRenderer {
     // temporary variable to save allocations
     private readonly m_tempScreenBox = new Math2D.Box();
 
+    private readonly m_renderer: THREE.WebGLRenderer;
+    private readonly m_layers: PoiLayer[] = [];
+
     /**
      * Create the `PoiRenderer` for the specified {@link MapView}.
      *
      * @param mapView - The MapView to be rendered to.
-     * @param textCanvas - The [[TextCanvas]] this `PoiRenderer` is associated to. POIs are added to
-     * the different layers of this [[TextCanvas]] based on renderOrder.
      */
-    constructor(readonly mapView: MapView, readonly textCanvas: TextCanvas) {
-        this.m_poiBatchRegistry = new PoiBatchRegistry(mapView, textCanvas);
+    constructor(readonly mapView: MapView) {
+        this.m_renderer = mapView.renderer;
+        this.m_poiBatchRegistry = new PoiBatchRegistry(this);
+    }
+
+    get renderer(): THREE.WebGLRenderer {
```

Comment:
probably better to pass down the renderer capabilities to the poi batches.

Reply:
done